.github/ISSUE_TEMPLATE/90_tech-checklist.md

```diff
@@ -106,5 +106,5 @@ assignees: ''
 - [ ] Re-render [under-review.qmd](https://github.com/journalovi/journalovi.github.io/blob/main/under-review.qmd) on the JoVI website (See instructions in the source code). Ensure that the added paper is listed on the [Under Review](https://www.journalovi.org/under-review.html) page in the correct location.
 - [ ] When this checklist is complete, create the following issues and tag the relevant people (Github IDs are listed on [People page](https://www.journalovi.org/people.html)), then close this issue:
    - [ ] [Recruit Associate Editor](new?assignees=&labels=pre-review&projects=&template=91_tech_org-recruit_editor.md&title=%5BPRE-REVIEW%5D+Recruit+Associate+Editor) (assign to the OC of this paper) (**NOTE:** this part of the process may change as we evolve the Editor-Wrangler role)
-   - [ ] [Open Materials Review](new?assignees=&labels=review&projects=&template=92_tech_open-review.md&title=%5BREVIEW%5D+Open+Materials+Review) (assign to the Open Practices Chair)
+   - [ ] [Open Materials Review](new?assignees=&labels=review&projects=&template=92_tech_open-review.md&title=%5BREVIEW%5D+Open+Materials+Review) (assign to the Open Practices Chair with the role "Write" on the repo)
```

Comment:
Would the open practices chair need write permissions on the repo? I'm not sure it makes sense to give them that permission.

Reply:
They should be able to close the issue by themselves. If other roles permit this, feel free to change.